go-controller/pkg/node/gateway_localnet_linux_test.go

```diff
@@ -285,10 +283,8 @@ var _ = Describe("Node Operations", func() {
 		app.Name = "test"
 		app.Flags = config.Flags
 		fExec = ovntest.NewFakeExec()
-		fakeOvnNode = NewFakeOVNNode(fExec)
-		fakeOvnNode.fakeExec.AddFakeCmd(&ovntest.ExpectedCmd{
-			Cmd: "ovs-vsctl --timeout=15 --no-heading --data=bare --format=csv --columns name list interface",
-		})
+		err := util.SetExec(fExec)
```

Comment:
seem like we can remove `FakeOVNNode` related functions in `ovn_test.go` as it is not used anymore by tests.

Reply:
True, it is unused. But it is an exported utility so in terms of linter that's ok. I decided to let it be as the utility is similar to the one we have on ovn package and I think it might be useful in the future..